app/src/main/java/com/adc/gpai/home/AdvisorScreen.kt

```diff
@@ -1,30 +1,137 @@
 package com.adc.gpai.home
 
+import android.app.Activity
+import android.content.ActivityNotFoundException
+import android.content.Intent
+import android.speech.RecognizerIntent
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Icon
 import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
 import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.adc.gpai.R
 import com.adc.gpai.ui.theme.GPAiTheme
 
 @Composable
 fun AdvisorScreen() {
     Column(
         modifier = Modifier
             .fillMaxSize()
-            .testTag("advisor_screen")
-        ,
+            .padding(8.dp)
+            .testTag("advisor_screen"),
         horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.Center
+        verticalArrangement = Arrangement.Bottom
     ) {
-        Text(text = "Advisor")
+        ChatInput()
     }
 }
 
+@Composable
+fun ChatInput(sendText: (String) -> Unit = {}) {
```

Comment:
Looks good! I'd just add some padding to the sides so it matches the width of the home toggle button

Reply:
I'd also add a modifier parameter here so that you can modify this ChatInput field more easily